src/DotLiquid.Tests/Util/StrFTimeTests.cs

```diff
@@ -26,7 +26,7 @@ public class StrFTimeTests
         [TestCase("%p", Result = "PM")]
         [TestCase("%S", Result = "14")]
         [TestCase("%U", Result = "02")]
-        [TestCase("%W", Result = "01")]
+        [TestCase("%W", Result = "02")]

```

Comment:
I'm working on reverting that change on my side, but this unit test is failing on my machine, which is why I made the change to begin with. I hadn't realized there was a CI server that was validating this but it's very odd that I'm getting an incorrect result on my system. I double checked the documentation on MSDN and the date itself. The correct result is in fact 01 but I can't explain why my system indicates that it's 02.